src/cable/connection.cr

```diff
@@ -167,10 +167,10 @@ module Cable
 
     private def subscribe_to_internal_channel
       server = Cable.server
-      channel = self
+      conn = self
       spawn(name: "Cable::Connection - subscribe_to_internal_channel") do
-        if !channel.connection_rejected?
-          server.backend.open_subscribe_connection(internal_channel)
+        if !conn.closed? || !conn.connection_rejected?
+          server.backend.subscribe(internal_channel)
```

Comment:
I originally wrote `open_subscribe_connection` thinking each new subscription was adding its own. It turns out that the Redis shard uses a single `Subscription` class and it [tracks all of the subscriptions](https://github.com/jgaskins/redis/blob/8d177adb4197c11fab7a07f8bf8dce2a98988f01/src/connection.cr#L465) so we can just add a subscription in to that.

Reply:
There was an issue where each time a user would connect to cable, Crystal would start a Fiber with this, but never reap that Fiber. If a user just refreshed the page, each time would add a new Fiber. Crystal would never release the Fiber because this spawn was showing as still active. I'm pretty sure this was a big memory leak. 

I used https://github.com/crystal-lang/perf-tools to inspect the active Fibers in the app and prove this count was increasing. By switching to this method, the Fibers are released almost immediately after.